qcodes/instrument/parameter.py

```diff
@@ -2075,40 +2132,48 @@ def _multiplier(self, multiplier: Union[int, float, Parameter]):
 
     # Division of the scaler
     @property
-    def division(self):
+    def division(self) -> float:  # type: ignore[return]
```

Comment:
why ignore return if the type is specified via casting? or am i misunderstanding something?

Reply:
Due to the mypy bug mentioned above. Remember that unused ignores is an error, so you can be sure this is needed anyway

Reply:
That is if mypy ever resolves the bug this will be an error and all to do is to remove this ignore comment